pkg/common/config/config.go

```diff
@@ -47,6 +47,14 @@ const (
 
 	// DefaultSecretDirectory is the default path to the secrets directory.
 	DefaultSecretDirectory string = "/etc/cloud/secrets"
+
+	// IPv6Family string representation for IPv6
+	IPv6Family = "ipv6"
```

Comment:
Hi @dvonthenen,

Why are these three constants duplicated? Can they be centralized?

Reply:
They can be... others had opinions about not doing it. I don't care either way.